flake8 (pyflakes) hangs all vagrant machines, fails on travis-ci

pyflakes 2.1.0 is broken, causing lots of issues.

we need to downgrade if they don't release a fix ASAP.

https://github.com/PyCQA/pyflakes/issues/422

```
flake8 runtests: commands[0] | flake8 src scripts conftest.py
src/borg/testsuite/platform.py:197:24: F811 redefinition of unused 'free_pid' from line 13
src/borg/testsuite/benchmark.py:29:19: F811 redefinition of unused 'cmd' from line 13
src/borg/testsuite/benchmark.py:56:22: F811 redefinition of unused 'cmd' from line 13
src/borg/testsuite/benchmark.py:62:33: F811 redefinition of unused 'cmd' from line 13
src/borg/testsuite/benchmark.py:67:32: F811 redefinition of unused 'cmd' from line 13
src/borg/testsuite/benchmark.py:72:29: F811 redefinition of unused 'cmd' from line 13
src/borg/testsuite/benchmark.py:78:28: F811 redefinition of unused 'cmd' from line 13
src/borg/testsuite/benchmark.py:83:26: F811 redefinition of unused 'cmd' from line 13
src/borg/testsuite/benchmark.py:88:26: F811 redefinition of unused 'cmd' from line 13
src/borg/testsuite/benchmark.py:93:27: F811 redefinition of unused 'cmd' from line 13
src/borg/testsuite/benchmark.py:99:26: F811 redefinition of unused 'cmd' from line 13
src/borg/testsuite/archiver.py:70:17: F811 redefinition of unused 'borg' from line 33
src/borg/platform/__init__.py:19:5: F811 redefinition of unused 'process_alive' from line 14
src/borg/platform/__init__.py:19:5: F811 redefinition of unused 'local_pid_alive' from line 14
src/borg/platform/__init__.py:21:5: F811 redefinition of unused 'swidth' from line 13
src/borg/platform/__init__.py:27:5: F811 redefinition of unused 'listxattr' from line 9
src/borg/platform/__init__.py:27:5: F811 redefinition of unused 'getxattr' from line 9
src/borg/platform/__init__.py:27:5: F811 redefinition of unused 'setxattr' from line 9
src/borg/platform/__init__.py:28:5: F811 redefinition of unused 'acl_get' from line 10
src/borg/platform/__init__.py:28:5: F811 redefinition of unused 'acl_set' from line 10
src/borg/platform/__init__.py:29:5: F811 redefinition of unused 'set_flags' from line 11
src/borg/platform/__init__.py:29:5: F811 redefinition of unused 'get_flags' from line 11
src/borg/platform/__init__.py:30:5: F811 redefinition of unused 'SyncFile' from line 12
```

--- a/src/borg/testsuite/archiver.py
+++ b/src/borg/testsuite/archiver.py
@@ -2868,7 +2868,7 @@ id: 2 / e29442 3506da 4e1ea7 / 25f62a 5a3d41 - 02
 
     @requires_hardlinks
     @requires_gnutar
-    def test_extract_hardlinks(self):
+    def test_extract_hardlinks_tar(self):
         self._extract_hardlinks_setup()
         self.cmd('export-tar', self.repository_location + '::test', 'output.tar', 'input/dir1')
         with changedir('output'):

